benchmarks/sharded-bm/README.md

```diff
@@ -102,8 +102,16 @@ Debug UI works, just use the machine public IP.
 The first step is to create VMs to run the nodes.
 
 You should be able to use any valid forknet image with [forknet terraform recipes](https://docs.nearone.org/doc/mocknet-guide-7VnYUXjs2A).
-This [setup](https://github.com/Near-One/infra-ops/tree/tpsnet/provisioning/terraform/infra/network/mocknet/tpsnet) is an example infra suitable for the benchmark. You can do something similar and deploy it with:
+This [setup](https://github.com/Near-One/infra-ops/tree/tpsnet/provisioning/terraform/infra/network/mocknet/tpsnet) is an example infra suitable for the benchmark. You can make a copy then make the following modifications:
 
+1. in `main.tf`: update `unique_id` and `start_height`, then specify the number
+of nodes to start & their regions. Number of instances specified in terraform
+(when totalled across regions) _MUST_ match the `CASE` you intend to run (eg, 20
+CP + 1 RPC = 21) -- see below or [`cases`](https://github.com/near/nearcore/tree/master/benchmarks/sharded-bm/cases) Note all regions may not
+support starting all types of instances (you can check in the console)
+1. in `resources.tf` modify the bucket prefix to be unique (will store the terraform state)
```

Comment:
Did you mean 2.?

Reply:
in markdown you can use 1. for the number each time and when it renders it will use the correct index (so you don't have to change numbers on other lines if you add / remove items).